app/admin/RTMediaFormHandler.php

```diff
@@ -1,17 +1,25 @@
 <?php
-/*
+/**
  * To change this template, choose Tools | Templates
  * and open the template in the editor.
+ *
+ * @package    rtMedia
+ * @subpackage Admin
  */
 
 /**
- * Description of RTMediaFormHandler
+ * Class for different form elements.
  *
  * @author udit
  */
 class RTMediaFormHandler {
 
-	public static function selectBox( $args ) {
+	/**
+	 * Function to show Select box.
+	 *
+	 * @param array $args Arguments.
+	 */
+	public static function selectBox( $args ) { // @codingStandardsIgnoreLine
```

Comment:
Use `phpcs:ignore` instead of `@codingStandardsIgnoreLine` because its deprecated.